src/generators/java/mekanism/generators/common/tile/TileEntityGenerator.java

```diff
@@ -83,4 +83,7 @@ protected ISyncableData syncableMaxOutput() {
     public BasicEnergyContainer getEnergyContainer() {
         return energyContainer;
     }
-}
\ No newline at end of file
+
+    @ComputerMethod(methodDescription = "Get the amount of energy produced by this generator in the last tick.")
+    abstract FloatingLong getProductionRate();
```

Comment:
As CodeQL is warning please add `@Override` annotations to the implementers of this.